### PR TITLE
3.3.38: twitter login from the extension

### PR DESCRIPTION
**Pull request by 2is10**
@ahsu1230, you asked me to let you know if I restyled the login form. I did a little. Feel free to apply the same changes to the login form on kifi.com.

image

### CODE REVIEW

File: packrat/styles/iframes/login.css [hunk 1]
```diff
@@ -151,15 +173,21 @@ input[type=password] {
...
   width: 12px;
   height: 16px;
   background-size: contain;
   background-image: url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAwAAAAQCAYAAAAiYZ4HAAAAeUlEQVR42sXRsQ2AMAxEUc+QdcgIsEooWYC50sAqWQAqKnPFgSyEpYSG4hW29SuLqt5KKQNk2ClDD3IRM8ygjvkZ9DwckCBQ4k5hsEHmcuRsTbxlG2xchpcg8LbZQEkcSt+CFbTSwrqeF0SqDoR+DqwInRu0aP3DegJzDobNoXlc6AAAAABJRU5ErkJggg==);
+  cursor: text;
+}
+@media (min-resolution: 1.3dppx) {
+.password-label::after {
+  background-image: url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABgAAAAgCAYAAAAIXrg4AAABMUlEQVR42u1VsUrEQBR8IHiNH6DcR6hdvsFO8a7xfuhsRPwHG231T4JFilvIJgfb2mgVZ+AKGdaY1Q0iZGDIYzNvJpvdbKwPZVnue++XTdPcgxvU77i+gi8c4z1q7Cdo2/YcBhWMuj5SA+2FDUXXdXtoXLM5kWv22neA8Foak0KsD5juQpt27/0W1yKEcECyxtgd70X0l18uKAS1iBvgtOeBTqiREE+v2KJeifkbDUQWCzmm9nMvvFYx4YME3NhAUCu9j7HFdSIqEgIKeU3OFDrNlA+oqqqZbgxT6G6wRGj/FDC8v67rMyxKy8HM3NLbxDwr6W0sxuSfBuh0n51zRyTr7AE0th1YZw8wwRQwBfxum4YQDsf40J5ojANszvofHBWZKMd1dm6NPwUWI5h7en8ANarzKGKRlt4AAAAASUVORK5CYII=);
+}
```
Comment (2is10) [inline packrat/styles/iframes/login.css:190]: This and the other 2x `image/png` files below I just moved up to be closer to the 1x versions.

File: packrat/styles/iframes/login.css [hunk 2]
```diff
@@ -98,6 +98,15 @@ input[type=password] {
...
 }
+.form-network.twitter {
+  background: #55acee;
+}
+.form-network.twitter:hover {
+  background: #66b4ef;
+}
+.form-network.twitter:active {
+  background: #52a7e7;
+}
```
Comment (2is10) [inline packrat/styles/iframes/login.css:109]: @ahsu1230 I made the `:active` color for the Twitter button a little darker, as the other buttons behave.

**Comment by ahsu1230** [comment]
awesome work! looks great. I'll mimic the same changes on the login page as well

**Comment by ahsu1230** [comment]
will this new extension be released immediately? If so, I should probably release twitter on the website as well.

**Comment by ahsu1230** [comment]
ah I saw your comment in the previous PR. In any case, I'll run it by PMs when to release twitter on the website. Thanks alot for taking this for me!

**Pull request merged**